cmake/tpls/DyninstCapstone.cmake

```diff
@@ -0,0 +1,69 @@
+#===========================================================
+#
+# Configure Capstone
+#
+#   ----------------------------------------
+#
+# Capstone_ROOT_DIR - Location of Capstone installation
+#
+# The individual find-modules use the <Package>_ROOT convention
+# as the first location to search for the package. If the user
+# specifies Capstone_ROOT_DIR, we override the <Package>_ROOT
+# values and require that each package ignores system directories.
+# In effect, this forces the package search to find only
+# candidates in <Package>_ROOT or CMAKE_PREFIX_PATH.
+#
+#===========================================================
+
+include_guard(GLOBAL)
+
+set(_min_version 6.0.0-Alpha1)
+
+if(Capstone_ROOT)
+  set(capstone_ROOT ${Capstone_ROOT})
+endif()
+
+if(capstone_ROOT_DIR)
+  set(Capstone_ROOT_DIR ${capstone_ROOT_DIR})
+  mark_as_advanced(Capstone_ROOT_DIR)
+endif()
+
+if(Capstone_ROOT_DIR)
+  set(capstone_ROOT ${Capstone_ROOT_DIR})
+  mark_as_advanced(CAPSTONE_ROOT)
+  set(capstone_DIR ${Capstone_ROOT_DIR})
+  mark_as_advanced(capstone_DIR)
+  set(_find_path_args NO_CMAKE_SYSTEM_PATH NO_SYSTEM_ENVIRONMENT_PATH)
+endif()
+
+find_package(capstone REQUIRED ${_find_path_args})
+
+# Only major versions of Capstone are compatible, but we can use anything newer than
+# the required minimum version.
+if(capstone_VERSION VERSION_LESS ${_min_version})
```

Comment:
The [docs](https://cmake.org/cmake/help/latest/command/if.html#version-less) says

```
Component-wise integer version number comparison (version format is
major[.minor[.patch[.tweak]]],  omitted components are treated as zero). Any non-integer
version component or non-integer trailing part of a version component effectively truncates
the string at that point.
```

Did this work for you?